apps/ui/src/networks/offchain/index.ts

```diff
@@ -49,7 +58,43 @@ export function createOffchainNetwork(networkId: NetworkID): Network {
     constants,
     helpers,
     actions: {
-      getVotingPower: () => Promise.resolve([])
+      getVotingPower: async (
+        strategiesAddresses: string[],
+        strategiesParams: any[],
+        strategiesMetadata: StrategyParsedMetadata[],
+        voterAddress: string,
+        block: number | null
+      ): Promise<VotingPower[]> => {
+        const result = await fetch(`${SCORE_URLS[networkId]}`, {
+          method: 'POST',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json'
+          },
+          body: JSON.stringify({
+            method: 'get_vp',
+            params: {
+              address: voterAddress,
+              space: '',
+              strategies: strategiesParams,
+              snapshot: block ?? 'latest'
+            }
+          })
+        });
+        const body = await result.json();
+
+        return body.result.vp_by_strategy.map((vp: number, index: number) => {
+          const strategy = strategiesParams[index];
+
+          return {
+            address: strategy.name,
+            value: BigInt(vp),
```

Comment:
`vp` here can be decimal number but we need to have it in base units (`BigInt(0.1)` throws).
For example see this space: http://localhost:8080/#/s-tn:weth.testsnap.eth/proposal/0xc827b7aad4e30ce7cee21bfb496228c7f2312937cecbcea605e79fd53d6f5361